src/sentry_value.c

```diff
@@ -1036,8 +1038,38 @@ sentry_value_new_breadcrumb(const char *type, const char *message)
     return rv;
 }
 
-void
-sentry_event_value_add_stacktrace(sentry_value_t event, void **ips, size_t len)
+sentry_value_t
+sentry_value_new_exception(const char *type, const char *value)
+{
+    sentry_value_t exc = sentry_value_new_object();
+    sentry_value_set_by_key(exc, "type", sentry_value_new_string(type));
+    sentry_value_set_by_key(exc, "value", sentry_value_new_string(value));
+    return exc;
+}
+
+sentry_value_t
+sentry_value_new_thread(uint64_t id, const char *name)
+{
+    sentry_value_t thread = sentry_value_new_object();
+
+    // TODO: would be nice to be able to actually use a `u64` as value.
+    char buf[100];
```

Comment:
The maximum length is 20 characters plus the zero byte.

```suggestion
    char buf[20 + 1];
```